tests/topotato/requirements.txt

```diff
@@ -0,0 +1,7 @@
+graphviz==0.20; python_version >= "3.8"
```

Comment:
Topotato does not use the python `graphviz` module, only the executable, so this is not needed.

Also, except pytest and scapy, topotato does not require specific versions.

Reply:
I removed it in new commits